README.md

```diff
@@ -10,11 +10,11 @@ female. All `names` are different. Key
 `wife`/`husband` can be either `None` or 
 name of another person.
 
-Create class `Person`. It's constructor takes
+Create class `Person`. It's `__init__` method takes
 and store `name`, `age` of a person.
 This class also should have a class attribute
 `people`, it is a dict that stores `Person` 
-instances by their `name`. Constructor should 
+instances by their `name`. The `__init__` method should 
```

Comment:
why did you change this file?

Reply:
i didnt change that. I think it it was IDE when i set up my this project. should i fix this?

Reply:
Yes, remove this file from the PR, please. Only `app/main.py` should be present.